src/components/Home/Home.js

```diff
@@ -172,33 +160,17 @@ const Main = styled.div`
   display: flex;
   height: 100%;
   flex-direction: column;
-  align-items: stretch;
-  justify-content: stretch;
 
   background-color: ${theme.mainBackground};
   background-image: url(${logo});
   background-position: 50% 50%;
   background-repeat: no-repeat;
 `
 
-const AppBarWrapper = styled.div`
-  flex-shrink: 0;
-`
-
-const ScrollWrapper = styled.div`
-  display: flex;
-  flex-direction: column;
-  justify-content: stretch;
-  overflow-y: scroll;
+const AppContent = styled.div`
   flex-grow: 1;
-`
-
-const AppWrapper = styled.div`
-  flex-grow: 1;
-  min-height: min-content;
-  display: flex;
-  align-items: stretch;
-  justify-content: space-between;
+  flex-shrink: 1;
+  min-height: 0;
```

Comment:
Hmm, I think we need to move `min-height` up to the `Main`, otherwise this can happen:

<img height="300" alt="screen shot 2018-11-14 at 2 26 11 pm" src="https://user-images.githubusercontent.com/4166642/48485948-77b5d880-e81a-11e8-8ac3-c6d1245df044.png">

I was able to fix the auto collapsing with

```js
const Main = styled.div`
  display: flex;
  height: 100%;
  min-height: 0;
  flex-direction: column;

  background-color: ${theme.mainBackground};
  background-image: url(${logo});
  background-position: 50% 50%;
  background-repeat: no-repeat;
`

const AppContent = styled.div`
  flex-grow: 1;
  flex-shrink: 1;
`
```

Reply:
I did it a bit differently to have the local scroll rather than full page scroll − `height: calc(100% - 54px)` on `AppContent`.

Reply:
D: it was a chrome flexbox issue?

Reply:
Yeah, one of these cases where Firefox and Chrome are handling flexbox differently :o